structuralcodes/codes/mc2010/_concrete_creep_and_shrinkage.py

```diff
@@ -275,7 +275,7 @@ def eps_cds0(
 
 def beta_ds(
     time: npt.ArrayLike, ts: float, notional_size: float
-) -> np.ndarray:
+) -> npt.ArrayLike:
```

Comment:
Shouldn't the output be the Union of float and ArrayLike?

Reply:
Fixed in ba81c03